mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -94,10 +102,14 @@ CLASS net/minecraft/class_1745 net/minecraft/entity/Entity
 		ARG 5 z
 		ARG 7 yaw
 		ARG 8 pitch
+	METHOD method_6921 rayTrace (DF)Lnet/minecraft/class_647;
 	METHOD method_6922 updateFall (DZLnet/minecraft/class_160;Lnet/minecraft/class_1372;)V
 		ARG 1 y
 		ARG 4 block
 		ARG 5 pos
+	METHOD method_6923 setBounds (FF)V
+		ARG 1 width
+		ARG 2 height
 	METHOD method_6927 setWorld (Lnet/minecraft/class_99;)V
 		ARG 1 world
 	METHOD method_6929 setBoundingBox (Lnet/minecraft/class_646;)V
```

Comment:
Might want to change this to setBounds now?

Reply:
what?

Reply:
that doesnt feel like a yarn name

Reply:
explain what's `feel like a yarn name`

Reply:
feels too shortened